docs/presentation/Content/RelatedPropertiesSpecification.md

```diff
@@ -1,60 +1,121 @@
-| `skipIfDuplicate`                  | No        | `boolean`                                                                           | `false`                        | Should this related properties specification be ignored if it duplicates another higher priority specification for the same relationship.                                                                                                                                                          |
+| Name                                                                              | Required? | Type                                                                                | Default             |
+| --------------------------------------------------------------------------------- | --------- | ----------------------------------------------------------------------------------- | ------------------- |
+| [`propertiesSource`](#attribute-propertiessource)                                 | Yes       | [`RelationshipPathSpecification`](../Common-Rules/RelationshipPathSpecification.md) |                     |
+| [`handleTargetClassPolymorphically`](#attribute-handletargetclasspolymorphically) | No        | `boolean`                                                                           | `false`             |
+| [`relationshipMeaning`](#attribute-relationshipmeaning)                           | No        | `"SameInstance" \| "RelatedInstance"`                                               | `"RelatedInstance"` |
+| [`properties`](#attribute-properties)                                             | No        | `Array<string \| PropertySpecification> \| "_none_" \| "*"`                         | `"*"`               |
+| [`autoExpand`](#attribute-autoexpand)                                             | No        | `boolean`                                                                           | `false`             |
+| [`skipIfDuplicate`](#attribute-skipifduplicate)                                   | No        | `boolean`                                                                           | `false`             |
+| [`nestedRelatedProperties`](#attribute-nestedrelatedproperties)                   | No        | [`RelatedPropertiesSpecification[]`](#related-properties-specification)             | `[]`                |
 
-### Relationship Meaning Attribute
+### Attribute: `propertiesSource`
 
-The attribute tells the presentation rules engine what the related properties mean to the instance whose properties are displayed. There are two possible options:
+[Specification of the relationship path](../Common-Rules/RelationshipPathSpecification.md) to follow when looking for related properties.
 
```

Comment:
This attribute seems to be similar to `relationshipPaths` of `RelatedInstanceNodesSpecification`. Could you adapt that description to suite this specification? Also please mention what happens when the relationships yield multiple output instances.

 > `Specifies a chain of [relationship path specifications](../Common-Rules/RelationshipPathSpecification.md) that forms a path from a source instance to the output instances. When this array is empty, the specification produces no results.`

Reply:
Updated, please have a look.

Reply:
This is much better now, thanks.